symsorter/src/app.rs

```diff
@@ -124,13 +121,15 @@ fn process_file(
         3 => 19,
         _ => 22,
     };
-    let archive = maybe_ignore_error!(Archive::parse(&bv));
+    let archive = maybe_ignore_error!(
+        Archive::parse(&bv).map_err(|e| anyhow!("failed to parse archive {}", e))
+    );
     let root = &RunConfig::get().output;
 
     for obj in archive.objects() {
-        let obj = maybe_ignore_error!(obj);
+        let obj = maybe_ignore_error!(obj.map_err(|e| anyhow!(e)));
```

Comment:
NB: We need to remember to come back to these and replace them with `.context("xyz")` once we get `symbolic` fixed.